source/virtualBuffers/__init__.py

```diff
@@ -671,10 +671,10 @@ def _getNearestTableCell(
 
 		# Cells are grouped by row, so in most cases, we simply need to search in the right direction.
 		for info in self._iterTableCells(tableID, direction=movement, startPos=startPos):
-			_ignore, row, col, rowSpan, colSpan = self._getTableCellCoords(info)
-			if row <= destRow < row + rowSpan and col <= destCol < col + colSpan:
+			cell = self._getTableCellCoords(info)
+			if cell.row <= destRow < cell.row + cell.rowSpan and cell.col <= destCol < cell.col + cell.colSpan:

```

Comment:
Could this line be split for readability

```suggestion
			if (
				cell.row <= destRow < (cell.row + cell.rowSpan)
				and cell.col <= destCol < (cell.col + cell.colSpan)
			):
```